### PR TITLE
Added Dockerfile and support for hosting frontend

### PR DESCRIPTION
**Pull request by benc-uk**
Dockerfile containerizes the PredictAPI and also acts as a server/host for the frontend. This makes the container nice and standalone.

I don't build model.zip as part of the Dockerfile but it could be added

### CODE REVIEW

**Comment by benc-uk** [comment]
Here it is running in ACI
http://romcom.northeurope.azurecontainer.io:5000/

The image on Dockerhub is `bencuk/romcom`

File: frontend/js/main.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ function submitScript(text) {
 }
 
 function checkGenre(text, callback) {
-    $.ajax("https://romcomnotromcom.azurewebsites.net/predict", {
+    console.log(`${API_BASE}/predict`)
```
Comment (jasoncabot-ms) [inline frontend/js/main.js]: Probably want to remove this `console.log`

Comment (benc-uk) [reply frontend/js/main.js]: Yeah, done

**Comment by jasoncabot-ms** [comment]
```
              |    |    |                 
             )_)  )_)  )_)              
            )___))___))___)\            
           )____)____)_____)\\
         _____|____|____|____\\\__
---------\                   /---------
  ^^^^^ ^^^^^^^^^^^^^^^^^^^^^
    ^^^^      ^^^^     ^^^    ^^
         ^^^^      ^^^
```

**Pull request merged**